pkg/cmd/roachtest/testdata/regression.diffs

```diff
@@ -162375,10 +162376,10 @@ diff -U3 --label=/mnt/data1/postgres/src/test/regress/expected/collate.windows.w
 +    b text COLLATE "C"
 +)
 +^
-+INSERT INTO collate_test1 VALUES (1, 'abc'), (2, '�bc'), (3, 'bbc'), (4, 'ABC');
++INSERT INTO collate_test1 VALUES (1, 'abc'), (2, '�bc'), (3, 'bbc'), (4, 'ABC');
```

Comment:
what changed to cause this diff?

Reply:
I'm not quite sure - perhaps the last time we simply downloaded artifacts from the failed nightly run and this time I manually kicked of a run. The difference seems to be with unicode characters. I reverted these changes.